packages/core/core/src/PackagerRunner.js

```diff
@@ -27,6 +29,14 @@ export default class PackagerRunner {
   }
 
   async writeBundle(bundle: Bundle) {
+    // deserialize asset graph from JSON
```

Comment:
Would be awesome to have a better way of serializing and deserializing complex objects from JSON like this...